waspc/cli/src/Wasp/Cli/Command/CreateNewProject/ProjectDescription.hs

```diff
@@ -0,0 +1,91 @@
+module Wasp.Cli.Command.CreateNewProject.ProjectDescription
+  ( initNewProjectDescription,
+    NewProjectDescription (..),
+  )
+where
+
+import Control.Monad (when)
+import Control.Monad.IO.Class (liftIO)
+import Data.List (intercalate)
+import Data.Maybe (isJust)
+import Path.IO (doesDirExist)
+import StrongPath.Path (toPathAbsDir)
+import Wasp.Analyzer.Parser (isValidWaspIdentifier)
+import Wasp.Cli.Command (Command)
+import Wasp.Cli.Command.Call (Arguments)
+import Wasp.Cli.Command.CreateNewProject.ArgumentsParser (NewProjectArgs (..), parseNewProjectArgs)
+import Wasp.Cli.Command.CreateNewProject.Common (getAbsoluteWaspProjectDir, throwInvalidTemplateNameUsedError, throwProjectCreationError)
+import Wasp.Cli.Command.CreateNewProject.Templates (StarterTemplateNames, getStarterTemplates, isValidTemplateName, templatesToList)
+import Wasp.Cli.Common (waspWarns)
+import qualified Wasp.Cli.Interactive as Interactive
+import Wasp.Cli.Terminal (asWaspWarningMessage)
+import Wasp.Util (indent, kebabToCamelCase)
+
+data NewProjectDescription = NewProjectDescription
+  { _projectName :: String,
+    _appName :: String,
+    _templateName :: Maybe String
+  }
+
+initNewProjectDescription :: Arguments -> Command NewProjectDescription
+initNewProjectDescription newArgs = do
+  newProjectArgs <- parseNewProjectArgs newArgs
+  templates <- liftIO getStarterTemplates
```

Comment:
Are these just template names? Here it looks like they are templates, but I realize from the createNewProjectDesription that it takes just names, so there are just names? Let's reflect that in the name? I guess the function `getStarterTemplates` should also reflect that in the name. I would probably rename it to `fetchStarterTemplateNames`.

Reply:
I thought it was clear from the type signature 
```hs
getStarterTemplates :: IO (Maybe StarterTemplateNames)
```
that it returned names 🤷 

I can adjust the name a bit more to make it more clear.